Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.yml

```diff
@@ -26,6 +26,14 @@ configuration:
   required: true
   section: Connect
   displaypassword: Password
+- display: Error on empty arguments
+  additionalinfo: If checked, will raise an error on empty arguments in the ad-get-user command.
```

Comment:
If checked, the command will raise an error if no argument is provided. If unchecked, the command will proceed with an empty argument.